credit_card.go

```diff
@@ -65,18 +65,23 @@ type CreditCardOptions struct {
 	UpdateExistingToken           string `xml:"update-existing-token,omitempty"`
 }
 
+// GetCustomerId gets the customer id of the credit card owner
 func (card *CreditCard) GetCustomerId() string {
 	return card.CustomerId
 }
 
+// GetToken gets the payment method token of the credit card
 func (card *CreditCard) GetToken() string {
 	return card.Token
 }
 
+// IsDefault returns whether the credit card is the default for the customer
 func (card *CreditCard) IsDefault() bool {
 	return card.Default
 }
 
+// GetImageURL gets a URL that points to a payment method
+// image resource (a PNG file) hosted by Braintree
```

Comment:
Can you remove the bracket text about it being a PNG, and replace the word `resource` with `file`?